### PR TITLE
Resolve source image by image manifest

### PR DESCRIPTION
**Pull request by tkdchen**
STONEBLD-1831

Source container image built by Konflux has tag in format
manifest_digest.src, where the manifest_digest is the binary image
manifest. This commit adds an additional resolver this kind of source
container images pulled from allowed registries.

### CODE REVIEW

**Comment by tkdchen** [comment]
Require #66 to be merged first.

**Review by chmeliik — APPROVED** [review]
LGTM after #66 is merged

**Comment by tkdchen** [comment]
Rebased. No change to the commit.

**Pull request merged**
